memory-viz/src/memory_model.ts

```diff
@@ -78,6 +72,40 @@ export class MemoryModel {
         this.roughjs_config = options.roughjs_config;
         this.rough_svg = rough.svg(this.svg, this.roughjs_config);
 
+        var styles = `
```

Comment:
Also, I would move this new code into a separate helper function in `style.ts`, replacing the old code.